django_spanner/__init__.py

```diff
@@ -34,22 +33,24 @@
 if django.VERSION[:2] == (3, 2):
     USING_DJANGO_3 = True
 
-if USING_DJANGO_3:
-    from django.db.models.fields import (
-        SmallAutoField,
-        BigAutoField,
-    )
-    from django.db.models import JSONField
+USING_DJANGO_4 = False
+if django.VERSION[:2] == (4, 2):
+    USING_DJANGO_4 = True
+
+from django.db.models.fields import (
+    SmallAutoField,
+    BigAutoField,
+)
+from django.db.models import JSONField
 
 __version__ = pkg_resources.get_distribution("django-google-spanner").version
 
 USE_EMULATOR = os.getenv("SPANNER_EMULATOR_HOST") is not None
 
 # Only active LTS django versions (2.2.*, 3.2.*) are supported by this library right now.
```

Comment:
the comment need to be updated accordingly

Reply:
Thanks, done